templates/definition/tariff/victron.yaml

```diff
@@ -0,0 +1,49 @@
+template: victron
+products:
+  - brand: Victron
+    description:
+      generic: VRM Solar Forecast
+requirements:
+  description:
+    en: "[vrm.victronenergy.com](https://vrm.victronenergy.com) get the 2-days-forecast from your installation in VRM portal. You need a free user access token."
+    de: "[vrm.victronenergy.com](https://vrm.victronenergy.com) abrufen der 2-Tage-Prognose einer Installation im VRM-Portal. Es wird ein kostenloses User Access Token benoetigt."
+  evcc: ["skiptest"]
+group: solar
+params:
+  - name: idsite
+    description:
+      en: VRM-ID of the installation
+      de: VRM-ID der Installation
+    help:
+      en: VRM-ID is shown as VRM Site ID in VRM general settings
+      de: VRM-ID wird in den allg. Einstellungen als VRM Site ID in VRM angezeigt
+    example: 123456
+    required: true
+  - name: token
+    description:
+      en: user access token - create in VRM
+      de: user access token - in VRM anlegen

```

Comment:
please split into description/help
Description describes what this values is. Help can give additional instruction on how to obtain it